src/qa-certification-workspace/tee-review-and-submit.service.ts

```diff
@@ -34,23 +36,35 @@ export class TeeReviewAndSubmitService {
     let repository;
     if (isWorkspace) {
       repository = withTransaction(this.workspaceRepository, trx);
-    } else {
-      repository = withTransaction(this.globalRepository, trx);
```

Comment:
This function takes a transaction, too, so it shouldn't be meddled with. Best to revert the changes in this file.